core/modules/remote/pubspec.yaml

```diff
@@ -4,34 +4,34 @@ publish_to: 'none'
 version: 0.0.1
 
 environment:
-  sdk: ">=3.1.5 <4.0.0"
-  flutter: "3.13.9"
+  sdk: ">=3.2.0 <4.0.0"
+  flutter: "3.16.9"
 
 dependencies:
-  collection: 1.17.2
-  dio: 4.0.6
+  collection: 1.18.0
+  dio: 5.0.0
   domain:
     path: ../domain
   flutter:
     sdk: flutter
   flutter_pretty_dio_logger: 2.0.3
   foundation:
     path: ../foundation
-  fpdart: 0.4.0
+  fpdart: 1.1.0
   freezed_annotation: 2.4.1
   get_it: 7.2.0
   json_annotation: 4.8.1
   json_serializable: 6.7.1
   project_linter:
     path: ../project_linter
-  retrofit: 3.3.1
+  retrofit: 4.1.0
 
 dev_dependencies:
   build_runner: 2.4.6
   freezed: 2.4.5
   mocktail: 0.3.0
```

Comment:
should be mocktail: 1.0.3

Reply:
check in other files also

Reply:
```
[presentation]: Try `flutter pub outdated` for more information.
[domain]: Because test >=1.25.0 depends on test_api 0.7.0 and mocktail 1.0.1 depends on test_api >=0.2.1 <0.7.0, test >=1.25.0 is incompatible with mocktail 1.0.1.
[domain]: So, because domain depends on both mocktail 1.0.1 and test 1.25.2, version solving failed.
[domain]: 
[domain]: 
[domain]: You can try one of the following suggestions to make the pubspec resolve:
[domain]: * Try upgrading your constraint on mocktail: flutter pub add dev:mocktail:^1.0.3
[domain]: * Consider downgrading your constraint on test: flutter pub add dev:test:^1.24.9
```